### DISCUSSION TITLE
Installation and Execution Issues with Fooocus on Intel Mac

### DISCUSSION THREAD

**Discussion by Anthony-qnn** [lllyasviel/Fooocus#1198] [Q&A]
Hello Fooocus Team,

I am reaching out because I have been trying to install and run Fooocus on my Intel Mac, but I am encountering persistent difficulties despite several troubleshooting attempts.

Following the installation instructions on your GitHub page, I've run into issues related to the typing_extensions and fastapi libraries. Specifically, the following error occurs when running python entry_with_update.py:

ImportError: cannot import name 'Doc' from 'typing_extensions' (/Users/anthonyquinchon/miniconda3/envs/fooocus/lib/python3.10/site-packages/typing_extensions.py)

I have attempted to update the concerned libraries and reinstall all project dependencies, but the error continues. I was wondering if you had any suggestions or additional steps I could follow to resolve this issue.

I am on macOS 14.1.2, with Python 3.10.13.

Thank you in advance for your help and guidance.

Best regards,
Anthony

**Comment by ebenverster** [comment]
I'm able to launch the application, but I get this when trying to generate an image:
<img width="1422" alt="Screenshot 2023-12-17 at 21 42 46" src="https://github.com/lllyasviel/Fooocus/assets/11445150/04ba4fc8-fe3f-4964-80da-4b39405f10f5">

<img width="1440" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/11445150/b192931c-ec79-469b-b158-dca304ef6668">

**Comment by lllyasviel** [comment]
hi currently intel mac is not supported.
the suppprt for mps m1 m2 is also experimental.
see also diffusionbee and drawthings for mac native image generator

**Comment by jm-berthelot** [comment]
Hello !

Just found a solution !

Here's a video tutorial : https://github.com/user-attachments/assets/8621ed43-a654-4977-b320-6cd08cf41aa1

And the link that the guy is talking about in the video : https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb#scrollTo=VjYy0F2gZIPR

Enjoy !

**Comment by lgwjames** [comment]
i have a intel mac and works to a point :)
